test/network/select_all_test.cpp

```diff
@@ -67,7 +67,7 @@ void *SelectAllTest(int port) {
     txn1.commit();
 
     pqxx::work txn2(C);
-    for (int i = 0; i < 2000; i++) {
+    for (int i = 0; i < 20; i++) {
```

Comment:
It occurs to me that this test is originally created to check whether the packet is sent out correctly when data size exceeds certain limit(#878), thus the data size was set large. It slows down the check phase.. Maybe we can later directly move the select all query check to the sql trace testing? Just as a reminder

Reply:
Sorrry, this looks like an unintended change. Will revert this.